# Description: Run ranlib before installing the library read-only
# Submitted by: antoine@ (r350207)
# TODO: Upstream

--- ./Makefile.pre.in.orig	2014-04-20 23:22:37.435954278 +1000
+++ ./Makefile.pre.in	2014-04-20 23:23:10.767071278 +1000
@@ -985,8 +985,8 @@
 			if test "$(SO)" = .dll; then \
 				$(INSTALL_DATA) $(LDLIBRARY) $(DESTDIR)$(LIBPL) ; \
 			else \
+				$(RANLIB) $(LIBRARY) ; \
 				$(INSTALL_DATA) $(LIBRARY) $(DESTDIR)$(LIBPL)/$(LIBRARY) ; \
-				$(RANLIB) $(DESTDIR)$(LIBPL)/$(LIBRARY) ; \
 			fi; \
 		else \
 			echo Skip install of $(LIBRARY) - use make frameworkinstall; \
